data/json/snippets/snippets.json

```diff
@@ -48,7 +48,9 @@
     "type": "snippet",
     "category": "charsheet",
     "text": [
-      { "id": "charsheet_1", "text": "It's a Dungeons & Dragons 6th Edition character sheet.  This one is for a fighter." },
+      { 
+        "id": "charsheet_1", 
+        "text": "It's a Dungeons & Dragons 6th Edition character sheet.  This one is for a fighter." },
```

Comment:
Looks like an accidental change?

Reply:
Oh yeah, I forgot about that. It was the only line formatted differently and my OCD kicked in and I changed it!
Is it bad if I leave it in?

Reply:
Oh my bad nevermind, it's linted like that! I'll change it back then!

Reply:
For the next time, use the JSON linter which formats everything automagically: https://dev.narc.ro/cataclysm/format.html